### DISCUSSION TITLE
I propose creating Transmission private and public chat channels where contributors can coordinate releases and users can support each other or just hang out to possibly become future contributors.

### DISCUSSION THREAD

**Discussion by Pentaphon** [transmission/transmission#6226] [General]
There was recently talk of a private contributors chat in https://github.com/transmission/transmission/pull/6201 to give the contributors a private space to coordinate releases. I think a private chat will benefit the project but I think it's also worth adding a public chat to whatever solution is agreed upon so that users can support each other in real time and also give users a place to hang out and possibly become future contributors! There's no such thing on https://transmissionbt.com/ at the moment and the forum is fine but not a real-time solution. 

To everybody reading this: if you could have a Transmission public chat channel, what do you prefer? IRC? XMPP? Discord? Matrix? Something else?

To the contributors: feel free to use this discussion to try and hash this out together using the feedback from users that drop by.

CC: @tearfur @nevack @mikedld @ckerr @Coeur @GaryElshaw

**Comment by Coeur** [comment]
I could eventually install Telegram. You wouldn't see me on IRC or Discord.

**Reply by Coeur** [reply]
Oh, what about Slack?

**Reply by tearfur** [reply]
> Oh, what about Slack?

That could work.

**Reply by GaryElshaw** [reply]
Fine by me.

**Reply by Coeur** [reply]
@nevack I saw your message. Unfortunately, I can't reply before this weekend.

**Comment by dantearmok** [comment]
XMPP or Discord.  The latter is becoming quite popular with other software products.

**Comment by Pentaphon** [comment]
3 of 6 contributors I pinged so far are in favor of Slack. Any progress on this, guys?

**Comment by Coeur** [comment]
Just for the record, another communication channel could be to make a team on:
https://github.com/orgs/transmission/teams/collaborators/teams

Then associate the team with a private repo+issue tracker, and we can participate to that issue tracker which would only be visible to that team of "recurring contributors".

**Reply by Coeur** [reply]
It seems that no one made it yet after all this time.
So, I just made a new org, with a new private repo, just for that goal, private discussions about Transmission:
https://github.com/swiftmission/transmission

CC: @tearfur @nevack @mikedld @ckerr @GaryElshaw @Pentaphon

**Reply by Pentaphon** [reply]
@Coeur thanks for the invite but since I am not a contributor, I politely declined it. I hope the private repo helps the contributors coordinate new releases more efficiently.

**Reply by Coeur** [reply]
Then make a pull request, and become a contributor. :)

**Reply by Pentaphon** [reply]
> So, I just made a new org, with a new private repo, just for that goal, private discussions about Transmission:

Is this actively being used now? I'd like to see all the contributors using something to get 4.1.0 to the finish line before the Fall.

**Reply by Coeur** [reply]
The place I made is not actively used. I didn't see the admins of Transmission joining. My idea wasn't good enough.

**Comment by Pentaphon** [comment]
Any progress now that we are well into February? Even if we don't get a public chat for users, I'd love to see something be created for contributors so 4.1.0 can be planned by everybody involved.
